internal/pkg/release/version.go

```diff
@@ -93,5 +95,8 @@ func (v VersionInfo) String() string {
 	sb.WriteString(" at ")
 	sb.WriteString(v.BuildTime.Format("2006-01-02 15:04:05 -0700 MST"))
 	sb.WriteString(")")
+	if v.FIPS {
+		sb.WriteString(" [FIPS]")
```

Comment:
I don't think Fleet Server will let us enroll with this, it is validating the version and it must conform the semver rules: https://github.com/elastic/fleet-server/blob/94956dfe5da5b8c5859e23127bb5f8de24619e13/internal/pkg/api/userAgent.go#L88C22-L88C32